week-1/mandatory/2-classes-db/task.md

```diff
@@ -7,8 +7,28 @@ Below you will find a set of tasks for you to complete to consolidate and extend
 To submit this homework write the correct commands for each question here:
 
 ```sql
-
-
+--1
+select room_no from rooms where rate>100;
+--2
+select * from reservations where  (checkin_date  between '2020-09-01' and '2020-09-30')and ( checkout_date- checkin_date >3);
+--3
+select * from customers where city like 'M%';
+--4
+INSERT INTO room_types VALUES ('PENTHOUSE',185.00);
+--5
+INSERT INTO rooms   select 501 as room_no, def_rate,'PENTHOUSE' as room_type,null as no_guest from room_types where room_type='PENTHOUSE';
+
+INSERT INTO rooms   select 502 as room_no, def_rate,'PENTHOUSE' as room_type,null as no_guest from room_types where room_type='PENTHOUSE';
```

Comment:
Interesting use of `INSERT ... SELECT ... `and well-formed - well done.  You could, of course, have simply used:
`INSERT INTO rooms (room_no, rate, room_type) VALUES (501, 185, 'PENTHOUSE');`